### DISCUSSION TITLE
Working folder for Freesurfer

### DISCUSSION THREAD

**Discussion by mgreen-neura** [neurodesk/neurodesk.github.io-hugo#255] [General]
I see that dragging and dropping to the Nectar Virtual Desktop of NeuroDesk uploads my data uploaded to the "thinclient_drives/GUACFS" folder.

Is it reasonable to continue working from in this folder and creating outputs here? Or should I move it to "Desktop/neurodesktop-storage", say?

I've noticed that if I move data to "Desktop/neurodesktop-storage", it's not visible when using Freesurfer but navigating to the data in "thinclient_drives/GUACFS" is not a problem. See screenshot.

![image](https://user-images.githubusercontent.com/68717333/220808808-09294de3-e646-40ce-a6fd-ffa3f88462f8.png)

**Comment by civier** [comment]
Hi @mgreen-neura 

On the Nectar Virtual Desktop you don't really need to use /neurodesktop-storage because Neurodesktop is not running within a container, so if possible, I recommend to simply save your data in /home/vdiuser.

To my understanding, thinclient_drives/GUACFS is sitting on a different filesystem than the rest of the files in your virtual machine and I'm not sure about its capacity and speed -- so for the time being, I don't recommend using it other than for uploads. It is also important to notice that files saved to thinclient_drives/GUACFS/Download are automatically downloaded via the web browser and then DELETED from this folder, so you must not save there any files!

Good luck!
Oren

**Reply by mgreen-neura** [reply]
Thanks Oren, that's a big help. Much appreciated.

**Reply by aswinnarayanan** [reply]
@mgreen-neura, thanks for raising this, agreed with everything @civier's mentioned.

thinclient_drives/GUACFS is not suitable for processing, it has low read-write performance.
/home/vdiuser and /tmp should be accessible from the container, and provide full disk speed for processing data.

neurodesktop-storage is just an empty directory on nectar VDI, we might remove it in a future update

**Reply by mgreen-neura** [reply]
So I've run some data through Freesurfer in a folder in /home/vdiuser/data, say.

In order to download it back onto my local machine, do I need to transfer it first to the thinclient_drives/GUACFS ?
That appears to be the only "Shared Drive" available when I click ctrl-alt-shift to export the data.

![image](https://user-images.githubusercontent.com/68717333/221101982-7869c46c-604b-43a6-91a8-1d40c51fe438.png)

**Reply by stebo85** [reply]
If you copy data to thinclient_drives/GUACFS/Download it will start downloading the data 

A better option however is to install something like owncloud or Nextcloud to sync your data in and out and connect this with the storage your university offers

Happy to chat anytime and help set it up :)

**Reply by civier** [reply]
Hi @mgreen-neura 

Please consult the guide for the ARDC Virtual Desktop here:
https://tutorials.rc.nectar.org.au/virtual-desktop-service/03-general-tasks

Also, you can always easily navigate to this guide by going to ARDC virtual desktop main page: https://desktop.rc.nectar.org.au/
Click "?HELP" on the top, and then "this tutorial" link in the "SELF HELP" section.

BTW, do you want to save the data on your laptop/workstation, on a server behind a university firewall (i.e., requires VPN for access), or on a server with direct access from the internet?

Have a great weekend,
Oren
